NEWS

```diff
@@ -17,6 +17,7 @@
 * Fixes in RPM dependency on libibverbs
 * Fixes in ABI backward compatibility for active message protocol
 * Add support for DC full-handshake mode (off by default).
+* Fixes for handling a NULL cm on the ucp worker.
```

Comment:
`Fixes in TCP connection establishment (issue #6755)`

Reply:
why tcp..? the null cm can be any cm in the array (most likely rdmacm though) but it's ucp layer...

Reply:
How can we describe the error in more high-level in terms of how it affects a user?

Reply:
`Fixes for handling a missing sockaddr transport on a host` ?

Reply:
`Fixes for connection establishment protocol (tcpcm, rdmacm, etc.)` ?

Reply:
How about `Fixes for segmentation fault while listening for connections`